documentation/docs/getting_started.md

```diff
@@ -1,61 +1,53 @@
 ---
-description: Stronghold is an open-source software library can be used to protect any digital secret. It is a secure database for working with cryptography, which ensures that secrets are never revealed - but can be used according to best practices.
+description: Get started with Stronghold. Prerequisites, how to build and use the library.   
 image: /img/Banner/banner_stronghold.png
 keywords:
+
 - getting started
 - open-source
 - secure
 - secrets
 - Noise
 - database
 - p2p
----
-# Welcome
-
-![Stronghold](/img/Banner/banner_stronghold.png)
+- Rust
+- build
+- run
 
-Stronghold is an open-source software library that was originally built to protect IOTA Seeds, but can be used to protect any digital secret. 
-
-It is a secure database for working with cryptography, which ensures that secrets (like private keys) are never revealed - but can be used according to best practices.
-
-It provides its own peer-to-peer communication layer, so that different apps can securely communicate using the state-of-the-art Noise Protocol over libp2p. 
+---
 
+# Getting Started
 
-[![status](https://img.shields.io/badge/Status-Beta-green.svg)](https://github.com/iotaledger/stronghold.rs)
-![Audit](https://github.com/iotaledger/stronghold.rs/workflows/Audit/badge.svg?branch=dev)
-![Test](https://github.com/iotaledger/stronghold.rs/workflows/Test/badge.svg)
-[![docs](https://img.shields.io/badge/Docs-Official-green.svg)](https://stronghold.docs.iota.org)
-[![coverage](https://coveralls.io/repos/github/iotaledger/stronghold.rs/badge.svg?branch=dev)](https://coveralls.io/github/iotaledger/stronghold.rs?branch=dev)
-[![dependency status](https://deps.rs/repo/github/iotaledger/stronghold.rs/status.svg)](https://deps.rs/repo/github/iotaledger/stronghold.rs)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fiotaledger%2Fstronghold.rs.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fiotaledger%2Fstronghold.rs?ref=badge_shield)
+## Prerequisites
 
+To build Stronghold, you need a recent version of [Rust](https://www.rust-lang.org) installed.
 
-## 3rd Party Independent Security Audit
-In April of 2021, F-Secure performed a security assessment of the core crates of IOTA Stronghold and found nothing of concern. This is not an explicit declaration of fitness or freedom of error, but it is an indicator of the high quality of the code. You may review [the audit here](https://github.com/iotaledger/stronghold.rs/blob/dev/documentation/docs/meta/Audit.pdf).
+## Build the Library
 
-## Joining the discussion
-If you want to get involved in discussions about this library, or you're looking for support, go to the #stronghold-discussion channel on [Discord](https://discord.iota.org).
+The library comes with [examples](how_tos/cli/running_examples), but has no executables on its own. You can use the
```

Comment:
I can't put my finger on it, but somehow this section and the introduction of the cli example in the "Using the library" section seems weird to me. Maybe @aleksei-korolev has something to say about it? Maybe it is the repetition?